pages/governance/_id.vue

```diff
@@ -0,0 +1,146 @@
+<template>
```

Comment:
i think it should be `pages/proposals/_id.vue`?

Reply:
Agreed

Reply:
Renaming